code-of-conduct.md

```diff
@@ -3,38 +3,64 @@
 
 ## Acceptable Use of Company Hardware and Internet
+
 Computers and Internet are two of the most basic tools at our disposal as a company. We take pains to ensure that we have the best technology available to our team. With that in mind, we ask that employees use these tools respectfully and responsibly. Emails sent via any base2.io address should not contain any content that is deemed to be offensive including, but not restricted to, the use of vulgar or harassing language/images.
 
 ## Non-Discrimination and Anti-Harassment Policies
+
 The continued success of Base Two depends heavily on its effort to effectively utilize the talents of all qualified people. Therefore, Base Two prohibits employment decisions based on race, color, religion, sex, age, disability, national origin, ancestry, sexual orientation, gender identity, marital status, parental status, veteran status, military discharge status, or citizenship status, or any other protected status. All employment decisions continue to be based on individual merit, qualifications, and competence as they relate to a particular position and promotion of the principle of equal employment opportunity and affirmative action.
 
 ### How to Report an Issue
+
 If you, as a member of our team, feel uncomfortable for any reason you should feel empowered to speak to any other team member about the situation. If the issue stems from a specific person on the team and you feel comfortable talking through it with them directly, you should. If you do not feel comfortable doing that, you can bring your concerns to another member of the team and ask for their help in resolving the matter.
 
 ### If Someone Reports an Issue to You
+
 We are a very small team, and currently have no formal HR or Ethics group. Because of this, we rely on ourselves and each other to ensure that our workplace is safe and welcoming.
 
 In any instance where a team member comes to you with an issue, first ask the team member if they would like you to take action. If the answer is yes:
+
 - Commit to assisting in resolving their issue, or identify a person who can help them
 - Channel your childhood [peer mediation techniques](http://www.sese.org/wp-content/uploads/2013/08/Peer-Mediation-guide.pdf)
 - Act as an unbiased third party
 - Strive for confidentiality as much as possible
 
-We don't have much of a formal system yet, it's something we are continuing to create and refine as a team. If you have ideas, do not hesitate to share them.
+We don't have much of a formal system, it's something we are continuing to create and refine as a team. If you have ideas, do not hesitate to share them.
```

Comment:
💭 Possible re-word

```suggestion
Because we don't have a formal system, we're often refining this process as a team. If you have ideas, please share them.
```